recipe/meta.yaml

```diff
@@ -13,22 +13,23 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
   skip: true  # [osx]
+  skip: true  # [py<35]
```

Comment:
This needs a re-render as well.

Reply:
Ah, sorry. I got a little trigger happy with the merge. Just created an issue to re-render.

Reply:
No worries. Thanks for doing that. Looks ready to merge.

xref: https://github.com/conda-forge/openmc-feedstock/pull/23